src/WebJobs.Extensions.DocumentDB/DocumentDBAttribute.cs

```diff
@@ -0,0 +1,40 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+
+namespace Microsoft.Azure.WebJobs
+{
+    /// <summary>
+    /// Attribute used to binds a parameter to an DocumentDB type.
```

Comment:
Done.

---
In reply to: [56357111](https://github.com/Azure/azure-webjobs-sdk-extensions/pull/47#discussion_r56357111) [](ancestors = 56357111)